CONTRIB_pl.md

```diff
@@ -55,6 +55,39 @@ git version 2.17.1.windows.2
 
     ![pliki na dysku](docs/img/clone-fs.png)
 
+    * Dodaj upstream 
+        * przejdź do orginalnego źrudła i skopiuj link z ``Clone or download``. Następnie w cmd wpisz:
```

Comment:
```suggestion

```
Nie musimy mieć tego zdania, skoro poniżej i tak mamy poprawne wartości :) To może być mylące.